supertokens_python/types.py

```diff
@@ -61,3 +61,17 @@
     'required': ['supertokens', 'app_info', 'recipe_list', 'framework'],
     'additional_properties': False
 }
+
+
+class UsersResponse:
+    def __init__(self, users,
+                 next_pagination_token: Union[str, None]):
+        self.users = users
+        self.next_pagination_token = next_pagination_token
+
+
+class User:
```

Comment:
this user type is related to get users api response. the response also gives detail regarding the recipeId this user belongs to. So also add the information in this User class